JS_HW/task_4.js

```diff
@@ -0,0 +1,24 @@
+// Напишите функцию isEven(n), которая принимает один параметр, число, и
+// возвращает, четное ли оно.
+//  То есть функция может возвращать два значения: четное или нечетное.
+//     isEven(4); //true
+//     isEven(5); //false
+
+
+
+
+
+
+function isEven (number) {
+    if (number % 2 === 0) {
+        console.log ('true')
+        return true;
+    }
+
+    else {
```

Comment:
1. redundant `else`. you already have a return statement in `if` body above. 
2. for this task is enough to pass `return number % 2 === 0`, because this comparison return tru or false anyway